tl/features/context_match.py

```diff
@@ -38,7 +38,7 @@ def __init__(self, input_path, similarity_string_threshold, similarity_quantity_
         if pseudo_gt_column_name is not None and pseudo_gt_column_name in self.final_data.columns:
             self.final_data[pseudo_gt_column_name] = self.final_data[pseudo_gt_column_name].astype('float')
             self.final_data_subset = self.final_data[self.final_data[pseudo_gt_column_name] == 1]
-            self.to_result_data = self.final_data[self.final_data[pseudo_gt_column_name] == 0]
+            self.to_result_data = self.final_data[self.final_data[pseudo_gt_column_name] != 1]
```

Comment:
@HardiRathod why do you think this change is required?

Reply:
because pseudo_gt outputs 1 and -1 not 1 and 0.

Reply:
Done.